packages/web-wallet/src/hooks/useWebzjsActions.ts

```diff
@@ -18,17 +20,17 @@ export function useWebZjsActions(): WebzjsActions {
 
   const getAccountData = useCallback(async () => {
     try {
-      if (state.activeAccount !== undefined) {
-        return {
-          unifiedAddress: await state.webWallet!.get_current_address(
-            state.activeAccount,
-          ),
-        };
-      } else {
-        return {
-          unifiedAddress: await state.webWallet!.get_current_address(0),
-        };
-      }
+      const accountIndex = state.activeAccount ?? 0;
```

Comment:
I'm not sure about general activeAccount state in the web-wallet, why it is hardcocded to 0, and not throw error if its missing from state

Also in  `addNewAccountFromUfvk` with setting it upon install it defaults to hardcoded 0 instead of error handle
```
const account_id =
        (await state.webWallet?.create_account_ufvk(ufvk, birthdayHeight)) || 0;
      dispatch({ type: 'set-active-account', payload: account_id });
```